c/httpserver.c

```diff
@@ -3194,7 +3199,13 @@ static int handleHttpService(HttpServer *server,
 #endif
   if (request->authenticated == FALSE){
     /* could make this parameterizable */
-    respondWithError(response,401,"Not Authorized");
+    switch (racfRC) {
```

Comment:
It's best to check both SAF RC and RACF RC to determine the "expired" condition. If the RACF RC matches the value under a different SAF RC you will report the wrong error.